app/views/shared/_ssn_field.html.erb

```diff
@@ -6,9 +6,7 @@ locals:
 <%= f.label :ssn, class: 'usa-label' do %>
   <%= t('idv.form.ssn_label_html') %>
 <% end %>
-<div class="margin-bottom-1 usa-hint">
-  <%= t('forms.example') %> 123-45-6789
-</div>
+<%= f.hint t('forms.example') + ' 123-45-6789' %>
```

Comment:
same bit about maybe using a content_tag

```suggestion
<%= f.hint(content_tag(:span) do
  <%= t('forms.example') %> 123-45-6789
end) %>
```

Reply:
In the other one, I like how using the block makes it a bit clearer to mix strings and HTML. Since this one is a bit simpler as far as being a simple string concatenation (no HTML), I sort of prefer it as implemented already.